vehicles/src/modules/permit/dto/request/void-permit.dto.ts

```diff
@@ -21,18 +22,20 @@ export class VoidPermitDto {
     enum: ApplicationStatus,
     description: 'Revoke or void status for permit.',
     example: ApplicationStatus.REVOKED,
-    required: false,
-  })
+  })  
+  @IsEnum(ApplicationStatus)
```

Comment:
There's no change needed on the FE for this right? This field still represents whether or not to void or revoke?

Reply:
Yes. no changes to FE. Yes this field still represents whether or not to void or revoke.

Reply:
Ok got it, thanks